src/UI/components/ArchiveTab.tsx

```diff
@@ -84,11 +85,20 @@ export const ArchiveTab = () => {
         const path = await dialog.showOpenDialog({
             properties: ['openDirectory']
         });
-        console.log(path)
+
+        const pathname=path.filePaths[0];
+        console.log(pathname);
+      
         if (path.canceled) {
             console.log("User canceled the change path dialog.")
         }else{
             //TODO: take the last folder name and update it in UI
+            //check this method 
+            
+            var n = pathname.lastIndexOf("\\");
+            var result = pathname.substring(n + 1);
+            console.log(result)
```

Comment:
This logic will create issue in other platform because `\\` is windows thing. Mac and linux uses `/` in path. so if user is not on windows this will throw an error. 

if you want to get the file name from full path then you can use [path.basename()](https://nodejs.org/docs/latest-v14.x/api/path.html#path_path_basename_path_ext) which is cross platform.

The only catch is it is a node function, It might not be available in browser/web. To resolve it, you can either send the path to main process via ipcrender or you can find a web package which can emulate the same basename function in browser.